docs/modules/servers/pages/distributed/operate/metrics.adoc

```diff
@@ -95,25 +105,35 @@ This is for instance how the JMAP dashboard looks like:
 
 image::metrics.png[metrics for the JMAP protocol request latencies]
 
-== Running and configuring Grafana
+== Running and configuring Prometheus/Grafana
 
-The following command allow you to run a fresh grafana server :
+The following docker-compose.yml file will allow you to run a fresh prometheus/grafana montoring stack:
 
 ....
-docker run -i -p 3000:3000 grafana/grafana
+version: '3'
+
+services:
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    ports:
+      - "3000:3000"
+   # volumes:
+    #  - ./conf/grafana/grafana.ini:/etc/grafana/grafana.ini
+    #  - ./conf/grafana/provisioning:/etc/grafana/provisioning
+
+  prometheus:
+    image: prom/prometheus:latest
+    restart: unless-stopped
+    ports:
+      - "9090:9090"
+    volumes: #Map your scrape config to prometheus container.
+      - ./conf/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
```

Comment:
Can you include sample prometheus.yml as well as an example?

Reply:
sure, no problem with that.